play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/Utils.scala

```diff
@@ -28,10 +28,14 @@ trait Utils extends UtilsSupport {
     case _                       => false
   }
 
-  private[views] def calculateAssetPath(path: Option[String], file: String): String =
+  private def assetsUrl(file: String, useRebrand: Boolean): String =
+    if (useRebrand) uk.gov.hmrc.hmrcfrontend.controllers.routes.Assets.at(s"govuk/rebrand/$file").url
+    else uk.gov.hmrc.hmrcfrontend.controllers.routes.Assets.at(s"govuk/$file").url
+
+  private[views] def calculateAssetPath(path: Option[String], file: String, useRebrand: Boolean): String =
     path
       .map(p => s"$p/$file")
```

Comment:
this is matching how govuk-frontend does it - where rebrand prefix is only added to default path, otherwise it's expected to be passed as part of assetPath param

Reply:
yes, the `assetsUrl` method is only used when path is `None`, so only changing the default value.
Extracted as separate private method to have logic more clear than trying to do everything in `.getOrElse`

Reply:
renamed to `defaultAssetsUrl` even if it's internal thing, but I think it's worth directly pointing this is the default url not a url in general